cmd/root.go

```diff
@@ -55,7 +56,13 @@ There will be a ClusterPolicyReport with results for cluster-wide resources.`,
 			if err != nil {
 				return err
 			}
-			policiesFetcher, err := policies.NewFetcher(kubewardenNamespace)
+			skippedNsCSV, err := cmd.Flags().GetString("skipped-ns")
+			if err != nil {
+				return err
+			}
+			skippedNs := strings.Split(skippedNsCSV, ",")
+
+			policiesFetcher, err := policies.NewFetcher(kubewardenNamespace, skippedNs)
```

Comment:
TODO: look back, can't we just add the kubewarden namespace to the list of the skipped ones?